content/applications/hr/payroll/contracts.rst

```diff
@@ -3,223 +3,360 @@ Contracts
    :align: center
    :alt: Enter a new line for each type of garnishment.
 
-Select the :guilabel:`Garnished Type` from the drop-down menu. Choose from:
+.. important::
+   When the total amount has been paid for the salary attachment, navigate to the individual salary
+   attachment and click the :guilabel:`Mark as Completed` button at the top of the form. This
+   changes the status to :guilabel:`Completed` and the garnishments are no longer taken out of the
+   employee's paychecks.
 
-- :guilabel:`Attachment of Salary`: Any payments taken out towards something that is *not* child
-  support. Typically any garnishments such as lawsuit payments, payments toward taxes owed, etc.
-- :guilabel:`Assignment of Salary`: Any deduction that is not required but voluntary, such as a
-  pre-tax allocation to a college savings account.
-- :guilabel:`Child Support`: Any payments taken out specifically for child support.
+Offers
+======
 
-Enter the start and end dates the entry applies to. Click on the drop-down menu under
-:guilabel:`From` and :guilabel:`To`, navigate to the correct month and year by using the
-:guilabel:`< > (arrow)` icons, then click on the :guilabel:`date`.
+Once a contract has been created and/or modified, the contract must be sent to the employee to be
+accepted and signed.
 
-Last, enter the :guilabel:`Amount` that each payslip pays towards the entry.
+Send an offer
+-------------
 
-To delete a line, click the :guilabel:`🗑️ (trash can)` icon at the end of the line.
+Open an individual contract by navigating to :menuselection:`Payroll app --> Contracts ->
+Contracts`, and click on a contract to open the contract form. Click on the :guilabel:`Generate
+Offer` button at the top of the page and a :guilabel:`Generate Simulation Link` pop-up appears.
+
+.. image:: contracts/send-contract.png
+   :align: center
+   :alt: Send the contract to the employee via one of the buttons.
 
-Save and send the contract
---------------------------
+The :guilabel:`Generate Simulation Link` pop-up form contains all the information pulled from the
+contract, including the :guilabel:`Contract Template`, :guilabel:`Job Position`, :guilabel:`Job
+Title`, :guilabel:`Department`, :guilabel:`Contract Start Date`, :guilabel:`Default Vehicle`,
+:guilabel:`Contract Type`, and the :guilabel:`Yearly Cost`.
 
-Once a contract has been created and/or modified, save the contract by clicking the :guilabel:`Save`
-button. Next, the contract must be sent to the employee to be signed.
+At the bottom of the pop-up is a :guilabel:`Link Expiration Date`. This is the timeframe that the
+contract offer is valid for. The default that is pre-populated is `30 days`, but can be modified.
 
-Click on one of the following buttons to send the contract to the employee:
+Click the :guilabel:`Send By Email` button and a different pop-up appears, using the send offer
+email template. Make any modifications to the email, and attach any additional documents needed,
```

Comment:
```suggestion
Click the :guilabel:`Send By Email` button and :guilabel:`Send Offer Email` template pop-up appears. Make any modifications to the email, and attach any additional documents needed,
```
I am not sure if this is the exact way you would want to phrase it, but I think the template name should have either a label or italics.